src/cbc_sdk/platform/alerts.py

```diff
@@ -1371,43 +1390,70 @@ def set_minimum_severity(self, severity):
         self._criteria["minimum_severity"] = severity
         return self
 
-    def set_threat_notes_present(self, is_present):
+    def set_threat_notes_present(self, is_present, exclude=False):
         """
         Restricts the alerts that this query is performed on to those with or without threat_notes.
 
         Args:
             is_present (bool): If true, returns alerts that have a note attached to the threat_id
-
+            exclude (bool): If true, will set is_present in the exclusions. Otherwise adds to criteria
         Returns:
             AlertSearchQuery: This instance.
         """
-        self._criteria["threat_notes_present"] = is_present
+        if not exclude:
+            self._criteria["threat_notes_present"] = is_present
+        else:
+            self._exclusions["threat_notes_present"] = is_present
         return self
 
-    def set_alert_notes_present(self, is_present):
+    def set_alert_notes_present(self, is_present, exclude=False):
         """
         Restricts the alerts that this query is performed on to those with or without notes.
 
         Args:
             is_present (bool): If true, returns alerts that have a note attached
+            exclude (bool): If true, will set is_present in the exclusions. Otherwise adds to criteria
 
         Returns:
             AlertSearchQuery: This instance.
         """
-        self._criteria["alert_notes_present"] = is_present
+        if not exclude:
+            self._criteria["alert_notes_present"] = is_present
+        else:
+            self._exclusions["alert_notes_present"] = is_present
         return self
 
-    def set_remote_is_private(self, is_private):
+    def set_remote_is_private(self, is_private, exclude=False):
         """
         Restricts the alerts that this query is performed on based on matching the remote_is_private field.
 
-        This field is only present on CONTAINER_RUNTIME alerts and so filtering will be ingored on other alert types.
+        This field is only present on CONTAINER_RUNTIME alerts and so filtering will be ignored on other alert types.
 
         Args:
             is_private (boolean): Whether the remote information is private: true or false
+            exclude (bool): If true, will set is_present in the exclusions. Otherwise adds to criteria
+
+        Returns:
+            AlertSearchQuery: This instance.
+        """
+        if not exclude:
+            self._criteria["remote_is_private"] = is_private
+        else:
+            self._exclusions["remote_is_private"] = is_private
+        return self
+
+    def add_exclusion_remote_is_private(self, is_private):
```

Comment:
It looks like this one was not deleted when you made the change for `set_remote_is_private`

Reply:
thanks.  removed.